src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -495,6 +495,59 @@
   - name: Create a Windows task from a public GitHub repository which builds the Azure Container Builder image on Amd64 architecture.
     text: >
         az acr task create -t acb:{{.Run.ID}} -n acb-win -r MyRegistry -c https://github.com/Azure/acr-builder.git -f Windows.Dockerfile --commit-trigger-enabled false --pull-request-trigger-enabled false --platform Windows/amd64
+  - name: Create a Linux task from a public GitHub repository which builds the hello-world image with a git commit trigger and with the system-assigned managed identity
+    text: >
+        az acr task create -t hello-world:{{.Run.ID}} -n hello-world -r MyRegistry -c https://github.com/Azure-Samples/acr-build-helloworld-node.git -f Dockerfile --assign-identity
```

Comment:
nit: add an ending period, same for others.

Reply:
can we consistently write [system] for system identities? or is this a pattern followed by other RPs as well? Cuz I see an example below has to mention [system] bc it involves multiple identities.

Reply:
`--assign-identity` is default to mean the system-assigned identity for the task resource. The reason the `[system]` is used is for the case where the user wants to use the `SystemAssigned, UserAssigned` type for assigning multiple identities at once. This is an established precedent of other services, such as Azure VM: https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/qs-configure-cli-windows-vm#system-assigned-managed-identity